package.json

```diff
@@ -1,23 +1,24 @@
 {
   "devDependencies": {
-    "eslint": "^8.23.1",
+    "@babel/core": "^7.19.3",
+    "@babel/preset-env": "^7.19.3",
+    "@ngneat/falso": "^6.1.0",
+    "babel-jest": "^29.1.2",
+    "eslint": "8.22.0",
     "husky": "^8.0.1",
+    "jest": "^29.1.2",
     "prettier": "2.7.1"
   },
   "scripts": {
+    "test": "jest",
     "prepare": "husky install",
```

Comment:
I didn't find husky config. Did you miss it?

Reply:
When I will merge this brach, I will tune husky correctly.